update build scripts to use py3 container for running precheckin tox tests [cherry-pick]

wolverineav opened the pull request:
Reviewer: trivial

 - cherry pick to queens as well.

wolverineav committed:
update build scripts to use py3 container for running precheckin tox tests

bsn-bot3 commented:
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/openstack_horizon_bsn_pull_req/188/

wolverineav committed:
use sudo when doing a git clean

 - because container creates .tox directory as root user, outside
   the container, cleaning requires sudo

bsn-bot3 commented:
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/openstack_horizon_bsn_pull_req/190/
